tmt/convert.py

```diff
@@ -243,6 +243,21 @@ def read(path, makefile, nitrate, purpose, disabled, types):
             makefile, flags=re.MULTILINE)
         makefile = re.sub('.*rhts-lint.*', '', makefile)
 
+        # Detect framework
+        test_script = \
+            re.search(r'^run:.*\n\t\.\/(.*)$', makefile, re.M).group(1)
```

Comment:
What if there is no match? `re.search` would return None which would result in a traceback. Same below.